spec/support/factory_girl.rb

```diff
@@ -2,9 +2,15 @@
 
 FactoryGirl.define do
   factory :user do
-    sequence(:email) {|n| "user#{n}@example.com" }
+    sequence(:email) { |n| "user#{n}@example.com" }
     password 'password'
     password_confirmation 'password'
   end
 
+  factory :meetup do
+    sequence(:title) { |n| "This is meetup number #{n}." }
+    description "Let's see if we can get Dominos to deliver to another Dominos!"
+    user
+    datetime DateTime.now
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.